frontend/src/components/news-page/News.tsx

```diff
@@ -1,3 +1,43 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 
-export const News: React.FunctionComponent = () => <></>;
+export const News: React.FunctionComponent = () => {
+  const [articlesList, setArticlesList] = useState<any[]>([]);
```

Comment:
Don't use `any`! Please define an interface for this type, ideally in a separate file in `src/clients/redditApiClient.ts` along with the below